src/testIndex.ts

```diff
@@ -0,0 +1,11 @@
+const favoriteFruits: string[] = ['apples', 'strawberry', 'orange'];
+function addFruit(fruit: string): void {
```

Comment:
the same here
you don't have to specify `void` return type - if there is no type annotation and no return value, function will get `void` return type automatically

P.S. I won't comment further regarding inference, so get one more pass and try to find and remove it by yourself